packages/app/src/components/search/SearchModal.tsx

```diff
@@ -80,13 +80,29 @@ export const SearchModal = ({ toggleModal }: { toggleModal: () => void }) => {
   const { focusContent } = useContent();
   const { transitions } = useTheme();
 
-  const handleResultClick = () => {
-    toggleModal();
+  const isOpenNewTabOrWindow = (
```

Comment:
Any reason not to import from your other definition of this?

Reply:
I was afraid that exporting that utility function on the search plugin package would be out of scope. Since it is a public package I'm a bit reticent to extend it's API. But if you are making the question I believe I was being unnecessarily scared.
Thanks for the comment, I will move that way than =]